app/api/events.py

```diff
@@ -128,19 +183,29 @@ def after_create_object(self, event, data, view_kwargs):
         uer = UsersEventsRoles(user, event, role)
         save_to_db(uer, 'Event Saved')
 
-    decorators = (accessible_events, )
+    def before_create_object(self, data, view_kwargs):
+        data['starts-at'] = data['starts-at'].replace(tzinfo=None)
```

Comment:
I think it will be ```starts_at``` and ```ends_at```. I tested it again and same issue occurred as
```{"errors": [{"status": 500, "source": "", "detail": "'starts-at'", "title": "Unknown error"}], "jsonapi": {"version": "1.0"}}```